champss/ps-processes/ps_processes/processes/ps_inject.py

```diff
@@ -177,6 +205,25 @@ def onewrap_deltaDM(self):
         deltaDM = 1 / (1.0 / FREQ_BOTTOM**2 - 1.0 / FREQ_TOP**2) / self.f / DM_CONSTANT
         return deltaDM
 
+    def smear_fft(self, scaled_fft):
+
+        quadratic_term = 1e-11 #at 600 MHz
```

Comment:
Where does this quadratic term come from?  It mainly matters that it works, but I'm struggling to reproduce this value so I think more context would help